modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/UninitializedOperationMessageProcessor.java

```diff
@@ -0,0 +1,266 @@
+import org.mule.runtime.api.metadata.MetadataResolvingException;
+import org.mule.runtime.api.metadata.descriptor.ComponentMetadataDescriptor;
+import org.mule.runtime.api.metadata.descriptor.TypeMetadataDescriptor;
+import org.mule.runtime.api.metadata.resolving.MetadataResult;
+import org.mule.runtime.api.value.Value;
+import org.mule.runtime.core.api.MuleContext;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.internal.policy.PolicyManager;
+import org.mule.runtime.extension.api.runtime.config.ConfigurationInstance;
+import org.mule.runtime.extension.api.runtime.config.ConfigurationProvider;
+import org.mule.runtime.extension.api.runtime.operation.ExecutionContext;
+import org.mule.runtime.extension.api.values.ValueResolvingException;
+import org.mule.runtime.module.extension.api.runtime.privileged.ExecutionContextAdapter;
+import org.mule.runtime.module.extension.internal.runtime.resolver.ParameterValueResolver;
+import org.mule.runtime.module.extension.internal.runtime.resolver.ValueResolver;
+import org.mule.runtime.module.extension.internal.runtime.resolver.ValueResolvingContext;
+import org.mule.runtime.module.extension.internal.util.ReflectionCache;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+import javax.xml.namespace.QName;
+
+import com.google.common.collect.ImmutableMap;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Mono;
+
+public class UninitializedOperationMessageProcessor extends OperationMessageProcessor {
+
+  private OperationMessageProcessor operationMessageProcessorDelegate;
+  private Map<String, ValueResolver> resolvedParameters;
+
+  public UninitializedOperationMessageProcessor(ExtensionModel extension,
+                                                OperationModel operation,
+                                                PolicyManager policyManager,
+                                                Registry registry,
+                                                OperationMessageProcessor operationMessageProcessorDelegate,
+                                                Map<String, ValueResolver> resolvedParameters,
+                                                MuleContext muleContext,
+                                                ConfigurationProvider configurationProvider) {
+
+    super(extension,
+          operation,
+          configurationProvider,
+          operationMessageProcessorDelegate.target,
+          operationMessageProcessorDelegate.targetValue,
+          operationMessageProcessorDelegate.resolverSet,
+          null,
+          operationMessageProcessorDelegate.retryPolicyTemplate,
+          muleContext.getExtensionManager(),
+          policyManager,
+          registry.lookupByType(ReflectionCache.class).get());
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3655))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7719&issues=AWk7fTH-otIXnmsUoDr6&open=AWk7fTH-otIXnmsUoDr6)

[ISSUE_KEY]: AWk7fTH-otIXnmsUoDr6